frontend/components/commons/forms/questions-form/fields/text-area/TextArea.component.vue

```diff
@@ -83,15 +73,12 @@ export default {
   },
   methods: {
     onChangeTextArea(newText) {
-      this.$emit("on-change-text-area", [
-        {
-          id: this.optionId,
-          text: newText,
-          value: newText.length ? newText : null,
-        },
-      ]);
-
       const isAnyText = newText?.length;
+      this.$emit("on-change-text-area", {
```

Comment:
If the event is related to a property, should be reflected in the name, `on-change-value` or `on-change-text`. In any case, this could be tackled using v-model props, right?